cmd/bridge/config/auth/authoptions.go

```diff
@@ -252,12 +252,12 @@ func (c *completedOptions) getAuthenticator(
 ) (auth.Authenticator, error) {
 
 	if c.AuthType == "disabled" {
-		klog.Warning("running with AUTHENTICATION DISABLED!")
-
 		if c.BearerToken == "" {
+			klog.Warning("console is disabled -- no authentication method configured")
```

Comment:
maybe underline that this is meant to be used for development only

Reply:
I wanted to leave this vague since disabled auth can also represent an intermediate state when external OIDC is not fully configured. I didn't want to log "dev purpose only" lines in this case.